packages/test/test-end-to-end-tests/src/test/sharedStringLoading.spec.ts

```diff
@@ -40,6 +36,9 @@ describe("SharedString", () => {
         };
         const text = "hello world";
         const documentId = createDocumentId();
+        const driver = getFluidTestDriver();
```

Comment:
Does TS know this is `ITestDriver`, or is it `any`?

Reply:
getFluidTestDriver() returns ITestDriver, so ts knows